tests/framework/Optimizers/GeneticAlgorithms/testGAMaxwRep.xml

```diff
@@ -80,7 +81,7 @@
       </GAparams>
 
       <convergence>
-        <objective>-1</objective>
+        <AHDp>0.1</AHDp>
```

Comment:
This is not in the manual. @Jimmy-INL

Reply:
Can this be removed from the test change, or should we put off the GA demonstration for another PR?

Reply:
on it

Reply:
can you please convert it back to <objective>-1</objective>? If the test is long reduce the <limit>

Reply:
This shouldn't affect the plotter at all. Sorry, I thought it is hanging because other tests are failing not because of this. My bad, I think I am losing my memory